src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -124,9 +125,9 @@ bool MulticopterLandDetector::update()
         			fabsf(_vehicleAttitude.yawspeed)   > _params.maxRotation;
 
 	// check if thrust output is minimal (about half of default)
-	bool minimalThrust = _actuators.control[3] <= _params.maxThrottle;
+	bool flightThrust = (_actuators.control[3] > _params.maxThrottle);
```

Comment:
Ah ok, since this has a "!" below, it's still correct.
Maybe you should change the comment above then...